project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -8,6 +8,11 @@
 include_once 'c4m_user.features.inc';
 
 /**
+ * Define administrator role ID.
+ */
+define('C4M_USER_ROLE_ID_ADMINISTRATOR', 4);
```

Comment:
@ybaras 
remove this.
Use `user_role_load_by_name()` wherever you need to get the _rid_